src/main/java/com/opencart/steps/LoginPageBL.java

```diff
@@ -49,29 +38,21 @@ public ForgottenPasswordPageBL clickOnForgottenPassword() {
         return new ForgottenPasswordPageBL();
     }
 
-    /**
-     * Method inputs email to email input field on the login page.
-     *
-     * @param email is email that will be put on the login page.
-     */
     private void inputEmail(String email) {
         loginPage.getEmailInput().clear();
         loginPage.getEmailInput().sendKeys(email);
     }
 
-    /**
-     * Method inputs password to password input field on the login page.
-     *
-     * @param password is password that will be put on the login page.
-     */
     private void inputPassword(String password) {
         loginPage.getPasswordInput().clear();
         loginPage.getPasswordInput().sendKeys(password);
     }
 
-    /**
-     * Method clicks on Login Button on the login page.
-     */
+    public HeaderPageBL clickOnYourStoreButton() {
+        loginPage.getYourStoreButton().click();
+        return new HeaderPageBL();
+    }
```

Comment:
I believe it is better to move it to HeaderPageBL, because you can click on YourStore button from any page, not only LoginPage